src/BSON/UTCDateTime.c

```diff
@@ -105,37 +105,37 @@ static bool php_phongo_utcdatetime_init_from_hash(php_phongo_utcdatetime_t *inte
 
 /* Initialize the object from the current time and return whether it was
  * successful. */
-static bool php_phongo_utcdatetime_init_from_current_time(php_phongo_utcdatetime_t *intern) /* {{{ */
+static bool php_phongo_utcdatetime_init_from_current_time(php_phongo_utcdatetime_t* intern) /* {{{ */
 {
 	int64_t        sec, usec;
 	struct timeval cur_time;
 
 	gettimeofday(&cur_time, NULL);
-	sec = cur_time.tv_sec;
+	sec  = cur_time.tv_sec;
 	usec = cur_time.tv_usec;
 
 	intern->milliseconds = (sec * 1000) + (usec / 1000);
-	intern->initialized = true;
+	intern->initialized  = true;
 
 	return true;
 } /* }}} */
 
 /* Initialize the object from a DateTime object and return whether it was
  * successful. */
-static bool php_phongo_utcdatetime_init_from_date(php_phongo_utcdatetime_t *intern, php_date_obj *datetime_obj) /* {{{ */
+static bool php_phongo_utcdatetime_init_from_date(php_phongo_utcdatetime_t* intern, php_date_obj* datetime_obj) /* {{{ */
 {
 	int64_t sec, usec;
 
 	/* The following assignments use the same logic as date_format() in php_date.c */
 	sec = datetime_obj->time->sse;
 #if PHP_VERSION_ID >= 70200
-	usec = (int64_t) floor(datetime_obj->time->us);
+	usec = (int64_t)floor(datetime_obj->time->us);
 #else
-	usec = (int64_t) floor(datetime_obj->time->f * 1000000 + 0.5);
+	usec                  = (int64_t)floor(datetime_obj->time->f * 1000000 + 0.5);
```

Comment:
Don't get this change either.

Reply:
This is very strange. It's one character off from the `intern->` assignments below.